src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.tsx

```diff
@@ -64,55 +65,65 @@ export const WaterfallTaskStatusIcon: React.VFC<WaterfallTaskStatusIconProps> =
         clearTimeout(timeout);
       }
     },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    []
+    [] // eslint-disable-line react-hooks/exhaustive-deps
   );
   return (
-    <Tooltip
-      align="top"
-      justify="middle"
-      popoverZIndex={zIndex.tooltip}
-      enabled={enabled}
-      trigger={
-        <IconWrapper
-          onMouseEnter={onMouseEnter}
-          onMouseLeave={onMouseLeave}
-          key={`task_${taskId}`}
-          aria-label={`${status} icon`}
-          to={getTaskRoute(taskId)}
-          onClick={() => {
-            sendEvent({ name: "Click task status icon", status });
-          }}
-          data-cy="waterfall-task-status-icon"
-        >
-          <TaskStatusWrapper data-task-icon={identifier}>
-            <TaskStatusIcon status={status} size={16} />
-          </TaskStatusWrapper>
-        </IconWrapper>
-      }
-      triggerEvent="hover"
-    >
-      <div data-cy="waterfall-task-status-icon-tooltip">
-        <TooltipTitle
-          data-cy="waterfall-task-status-icon-tooltip-title"
-          weight="medium"
-        >
-          {displayName} {timeTaken && `- ${msToDuration(timeTaken)}`}
-        </TooltipTitle>
-        {loading ? (
-          <Skeleton />
-        ) : (
-          <>
-            {testResults?.map(({ id, testFile }) => (
-              <TestName key={id}>{testFile}</TestName>
-            ))}
-            {failedTestDifference > 0 && (
-              <div>and {failedTestDifference} more</div>
-            )}
-          </>
-        )}
-      </div>
-    </Tooltip>
+    <>
+      {enabled && (
```

Comment:
enabled is toggled by a timeout so any change will be delayed. 

I tried to replace this with a non timed out state value and that was also slower then what we had before. Reacts Virtual dom is working against us in this case since there are so many elements on the screen. The approach we had before technically by passed it the virtual dom by injecting styles directly into the dom.